src/css/profile/wearable/changeable/theme-circle/theme.layout.less

```diff
@@ -4,7 +4,7 @@ Listview
 @list_item_max_height: 360 * @unit_base;
 @list_item_line_height: 73 * @unit_base;
 @list_item_anchor_padding: 9 * @unit_base 30 * @unit_base 8 * @unit_base;
-@font_size_list_item: 36 * @unit_base;
+@font_size_list_item: 38 * @px_base;
```

Comment:
size is ok but I think we should use unit_base according to this file.

Reply:
@grzegorz-czajkowski why? I talked about it on chat, and there was information that now we use `@px_base` and `@sp_base`. Is it true only for the mobile profile?

Reply:
Ok.

Reply:
I think we should define and use sp_base in case of fonts

Reply:
Please define and use sp_base according to conversation.

Reply:
Done!